src/main/java/edu/itu/cavabunga/rest/CalendarRestController.java

```diff
@@ -2,12 +2,17 @@
 
```

Comment:
In my pov, we dont need a rest package. We can move this controller to core with restcontroller name.